CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Changelog for NeoFS Monitor
 ### Added
 
 ### Changed
+- Application starts with only one alive connection (#181)
```

Comment:
It's more like `fixed`, it was always supposed to work like this. And it can be rephrased a bit better.